README.md

```diff
@@ -0,0 +1,146 @@
+# Capsule sandbox
+
+The Capsule sandbox is a containerized development environment for building Capsule applications in Rust.
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Getting Started](#getting-started)
+  * [Vagrant and VirtualBox](#vagrant-and-virtualbox)
+  * [Linux Distributions](#linux-distributions)
+  * [Without Docker](#without-docker)
+  * [Kernel NIC Interface](#kernel-nic-interface)
+* [Packaging for Release](#packaging-for-release)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Introduction
+
+`Capsule` is built on Intel's [Data Plane Development Kit](https://www.dpdk.org/) release 18.11. While it is written in Rust, it needs to be able to call `DPDK`'s C functions installed separately on the host as shared dynamic libaries. To make developing and running Capsule applications easy, we created the sandbox container with all the necessary dependencies. You can pull down the container and start writing network functions in stable `Rust` right away.
```

Comment:
when mentioning needs to able to cal..., do we want to link to the ffi build.rs in `capsule`?

Reply:
that may confuse people, not everyone does ffi. if people want to learn, they will find it.